x-pack/plugins/uptime/server/lib/helper/get_filtered_query_and_status.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get } from 'lodash';
+import { getFilteredQuery } from './get_filtered_query';
+
+const getMonitorsListFilteredQuery = (filters: any): string | undefined => {
```

Comment:
I would say this could be better documented, but since it should go away sometime in the near future, I'm fine with it as is.

Reply:
I can add comments explaining why these functions exist. ++

Reply:
Resolved in https://github.com/elastic/kibana/pull/29398/commits/c747e4f456ce1cae128f33edd7ffd26d1bdf53ef.